packages/tslint-config/src/rules/blueprintClassesConstantsRule.ts

```diff
@@ -78,10 +79,11 @@ function getAllMatches(className: string) {
 
 /** Produce replacement text for a string literal that contains invalid classes. */
 function getLiteralReplacement(className: string, ptClassStrings: string[]) {
-    // remove all illegal classnames, then slice off the quotes, and trim any remaining white space
+    // remove all illegal classnames, then slice off the quotes, then merge & trim any remaining white space
     const stringWithoutPtClasses = ptClassStrings
         .reduce((value, cssClass) => value.replace(cssClass, ""), className)
         .slice(1, -1)
+        .replace(/(\s)+/, "$1")
```

Comment:
ah, that's beautiful

Reply:
yes!! sometimes regexes are just the best